howso/react_series.amlg

```diff
@@ -43,35 +43,60 @@
 	#react_series
 	(declare
 		(assoc
+			;{type "list" values "string"}
```

Comment:
missing some parameters such as 'exclude_novel_nominals_from_uniqueness_check'.  I suggest cross-referencing howso-engine-py's core.py calls to make sure whatever parameters are being passed in there are also specified here